test/integration_config.json

```diff
@@ -1,66 +1,76 @@
 [
-  { "language": "gomodule", "version": "1.18", "port": "1323", "repo": "gambtho/go_echo" },
+  { "language": "gomodule", "version": "1.18", "port": "1323", "serviceport": 80, "repo": "gambtho/go_echo" },
```

Comment:
nit: can we reformat this one to be multiple lines like all the other objects in this json file?

Reply:
yep, just changed it